dependency/health_service_test.go

```diff
@@ -230,9 +248,7 @@ func TestNewHealthServiceQuery(t *testing.T) {
 		tc := test.(testCase)
 		t.Run(fmt.Sprintf("%d_%s", i, tc.name), func(t *testing.T) {
 			act, err := NewHealthServiceQuery(tc.i)
-			if (err != nil) != tc.err {
-				t.Fatal(err)
-			}
+			assert.Equal(t, tc.err, err)
```

Comment:
thanks for making this change, makes a lot of sense to validate the actual error text